src/docs/asciidoc/en/firebirddocs/gfix/firebird-gfix.adoc

```diff
@@ -1445,13 +1470,45 @@ If you set the database to `async` mode (forced writes disabled) then it is poss
 
 [WARNING]
 ====
-If your database was originally created with Interbase 6 or an early beta version of Firebird then the database will be running in asynchronous mode -- which is not ideal.
+If your database was originally created with Interbase 6 or an early beta version of Firebird, then the database will be running in asynchronous mode -- which is not ideal.
 ====
 
+[[gfix-nolinger]]
+== Close Database Ignoring Linger Setting
+
+Firebird 3 introduced the database "`linger`" capability, which allows the SuperServer database engine to keep a database file (and its page cache, etc) open for a configured number of seconds after the last connection was closed.
+This is generally beneficial for performance, but not ideal for situations where a database needs a forced shutdown, and the file needs to be accessed by applications other than Firebird.
+
+When the database has no connections, the `-{wj}nol[inger]` option will close the database file immediately.
+The `LINGER` setting of the database is retained and will work normally for the next connection.
+
+[listing,subs=+quotes]
+----
+gfix -nol[inger] _database_name_
+----
+
+NOTE: The `-{wj}nol[inger] option was introduced in Firebird 3.
+
+[[gfix-icu]]
+== Rebuild ICU dependencies
+
+Firebird uses the ICU library for some of its collations, and for indices on fields using those collations.
+When moving a database between different Firebird installations -- without using backup and restore -- and the destination Firebird installation uses a different ICU version, querying on fields with those collations or using those indices will fail.
+Before Firebird 3, the only option was to backup and restore the database.
+
```

Comment:
For linux need to use -icu switch may also take place after regular OS upgrade which may change version of system ICU library. Also you may mention that the switch is 'safe' - if no action actually need to be taken gfix will do nothing and exit immediately.

Reply:
Thanks, will update later this week